lib/Runtime/Library/ConcatString.h

```diff
@@ -9,18 +9,30 @@ namespace Js
     class LiteralStringWithPropertyStringPtr : public LiteralString
     {
     private:
-        PropertyString * propertyString;
+        // TODO: Can we make PropertyString && LiteralStringWithPropertyStringPtr share the string buffer?
+        Field(PropertyString*) propertyString;
 
     public:
         PropertyString * GetPropertyString() const;
+        PropertyString * GetPropertyStringLazy(); // Get if it's there, otherwise bring it in.
```

Comment:
nit: this name is confusing to me. i think we could come up with something better, maybe `GetOrAddPropertyString`?

Reply:
I'm not strong on naming and what you suggest sounds good.